blueman/main/Manager.py

```diff
@@ -209,8 +209,11 @@ def toggle_trust(self, device):
     def send(self, device, f=None):
         adapter = self.List.Adapter
 
+        assert adapter
```

Comment:
We should just use self.List.adapter, having the ``adapter`` variable is a bit pointless. But why does the assert need to be here? We exited long before if we could not find an adapter.

Reply:
`self.List.Adapter` might be `None` so that the attribute getters used below would not be available and crash. `assert` just moves that crash here but also tells mypy that the value is not treated optional after this point.